plugins/bn-avro/qml/Keyboard_avro_emaill.qml

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
this filename has a spelling error as well it should be "email" not "emaill" (in addition to it needing to be bn-avro)